Cork/Views/Settings/Panes/General Pane.swift

```diff
@@ -75,10 +75,10 @@ struct GeneralPane: View
                 
                 LabeledContent {
                     Toggle(isOn: $showDescriptionsInSearchResults) {
-                        Text("Show descriptions in search results")
+                        Text("Show descriptions")
```

Comment:
Avoid repeating 'search results' twice on one line

Reply:
I would perhaps put "Show package descriptions" to make it more clear, but otherwise I agree

Reply:
I'll change this, that sounds better!